packages/ses-ava/src/ses-ava-test.js

```diff
@@ -61,16 +98,16 @@ const testerMethodsWhitelist = [
 
 /**
  * @param {TesterFunc} testerFunc
- * @param {Logger} [logger]
+ * @param {Logger} logger
  * @returns {TesterFunc} Not yet frozen!
  */
-const wrapTester = (testerFunc, logger = defaultLogger) => {
+const wrapTester = (testerFunc, logger) => {
   /** @type {TesterFunc} */
   const testerWrapper = (title, implFunc, ...otherArgs) => {
     /** @type {ImplFunc} */
-    const testFuncWrapper = t => {
-      harden(t);
-      return logErrorFirst(implFunc, [t, ...otherArgs], 'ava test', logger);
+    const testFuncWrapper = originalT => {
```

Comment:
Please disregard this comment: There’s this neologism “OG” which is the initialism for “Original Gangsta” and I think `ot` would be funny here.

Reply:
;)